### PR TITLE
Improvement/public model docs comments

### PR DESCRIPTION
**Pull request by patryk0493**

### CODE REVIEW

File: src/lib/CellTemplates/keyCodeCheckings.ts [hunk 1]
```diff
@@ -1,22 +1,62 @@
 import { keyCodes } from '../Functions/keyCodes';
 
+/**
+ * Checks that the pressed key's `keyCode` is is one of alphanumeric keys
```
Comment (myskamil) [inline src/lib/CellTemplates/keyCodeCheckings.ts]: * Checks that the pressed key's `keyCode` is one of alphanumeric keys

File: src/lib/Model/PublicModel.ts [hunk 2]
```diff
@@ -1,149 +1,429 @@
...
+/**
+ * `CellChange` type is used by `onCellsChanged`. It represents mutually exclusive changes on a single cell.
+ * 
+ * @see https://reactgrid.com/docs/3.1/7-api/1-types/2-cell-change/
+ */
 export type CellChange<TCell extends Cell = DefaultCellTypes & Cell> = TCell extends Cell ? Change<TCell> : never;
 
-// ASK ARCHITECT BEFORE INTRODUCING ANY CHANGE!
+/**
+ * `Change` interface represents a particullar change on a single cell on the cell template.
```
Comment (myskamil) [inline src/lib/Model/PublicModel.ts]: * `Change` interface represents a particular change on a single cell on the cell template.

File: src/lib/Model/PublicModel.ts [hunk 3]
```diff
@@ -1,149 +1,429 @@
...
+     */
     getClassName?(cell: Compatible<TCell>, isInEditMode: boolean): string;
-    // Render the cell content
-    render(cell: Compatible<TCell>, isInEditMode: boolean, onCellChanged: (cell: Compatible<TCell>, commit: boolean) => void): React.ReactNode;
+
+    /** 
+     * Renders the cell content
+     * 
+     * @param cell Incoming `Compatible` cell
+     * @param isInEditMode Flag is set to `true`,if cell is rendered in edit mode
```
Comment (myskamil) [inline src/lib/Model/PublicModel.ts]: isInEditMode Flag is set to `true`, if cell is rendered in edit mode

**Review by myskamil — CHANGES_REQUESTED** [review]
Looks very nice, just small mistakes 👍

**Review by patryk0493 — COMMENTED** [review]
Misspellings fixed

**Pull request merged**